apps/pie-storybook/stories/pie-radio.stories.ts

```diff
@@ -1,19 +1,80 @@
 import { html } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
 import { type Meta } from '@storybook/web-components';
 
 import '@justeattakeaway/pie-radio';
-import { type RadioProps } from '@justeattakeaway/pie-radio';
+import { type RadioProps as RadioBaseProps, defaultProps } from '@justeattakeaway/pie-radio';
+
+import { type SlottedComponentProps } from '../types';
 
 import { createStory } from '../utilities';
 
+type RadioProps = SlottedComponentProps<RadioBaseProps>;
 type RadioStoryMeta = Meta<RadioProps>;
 
-const defaultArgs: RadioProps = {};
+const defaultArgs: RadioProps = {
+    ...defaultProps,
+    slot: 'Label',
+    value: 'value',
+};
 
 const radioStoryMeta: RadioStoryMeta = {
     title: 'Radio',
     component: 'pie-radio',
-    argTypes: {},
+    argTypes: {
+        checked: {
+            description: 'The checked state of the radio.',
+            control: 'boolean',
+            defaultValue: {
+                summary: defaultArgs.checked,
+            },
+        },
+
+        defaultChecked: {
+            description: 'The default checked state of the radio (not necessarily the same as the current checked state). Used when the radio is part of a form that is reset.',
+            control: 'boolean',
+            defaultValue: {
+                summary: defaultArgs.defaultChecked,
+            },
+        },
+
+        disabled: {
+            description: 'Same as the HTML disabled attribute - indicates whether or not the radio is disabled.',
+            control: 'boolean',
+            defaultValue: {
+                summary: defaultArgs.disabled,
+            },
+        },
+
+        name: {
+            description: 'The name of the radio (used as a key/value pair with `value`). This is required in order to work properly with forms.',
+            control: 'text',
+            defaultValue: {
+                summary: defaultArgs.name,
+            },
+        },
+
+        required: {
+            description: 'Same as native required attribute. If any radio button in a same-named group of radio buttons has the required attribute, a radio button in that group must be checked, although it doesn\'t have to be the one with the attribute applied.',
+            control: 'boolean',
+            defaultValue: {
+                summary: defaultArgs.required,
+            },
+        },
+
+        slot: {
```

Comment:
Out of interest why are we opting for a slot here?

Reply:
This was mainly to match the checkbox component but maybe a prop would be better here.

Any strong feelings @dandel10n?

Reply:
For checkbox we went with a slot because in Figma component uses a Label child component that has its own options like trailing content etc. So to give consumers more freedom to customize the label we made it a slot.

For radio I don't see Label child component being used but in theory label can still have a link as part of it for example so slot makes it easier for consumers to implement.